docs/use/setup-wallet/index.md

```diff
@@ -97,38 +99,17 @@ Finally, to see an overview of your CFG accounts, click on "Receive" under your
 
```

Comment:
Line 80: Could be even deleted because the instruction to back up your seed phrase and wallet is already included in Talisman manual